### ISSUE TITLE
AttributeError: module 'posthog.queries.sessions' has no attribute 'Sessions'

### ISSUE THREAD

**Issue by kappa90** [PostHog/posthog#5025]
```
AttributeError: module 'posthog.queries.sessions' has no attribute 'Sessions'
(6 additional frame(s) were not displayed)
...
  File "posthog/api/dashboard.py", line 101, in get_items
    update_dashboard_items_cache(dashboard)
  File "posthog/tasks/update_cache.py", line 73, in update_dashboard_items_cache
    update_cache_item(cache_key, cache_type, payload)
  File "posthog/tasks/update_cache.py", line 64, in update_cache_item
    result = _calculate_by_filter(filter, key, team_id, cache_type)
  File "posthog/tasks/update_cache.py", line 144, in _calculate_by_filter
    insight_class = import_from(insight_class_path[0], insight_class_path[1])
  File "posthog/tasks/update_cache.py", line 132, in import_from
    return getattr(importlib.import_module(module), name)
```

**Comment by paolodamico** [comment]
For context here, I had a call with @kappa90 today where we tried to debug what's going on because when he presses refresh all in his dashboards, he'll get a 500 error (which is the error detailed above). As the individual dashboard refreshes are currently gone, there's no way to refresh their dashboards. 

This looks like it's a problem refreshing some sessions query, but we tried removing the sessions query from the dashboard and the error kept happening.

Could use some help from Core Analytics Team here. CC @EDsCODE @Twixes
